src/swapd/runtime.rs

```diff
@@ -1474,14 +1518,16 @@ impl Runtime {
                     }),
                 )?;
             }
-
```

Comment:
Unintended removal

Reply:
If this is the only thing remaining, I'll take care of it in a follow-up.